pom.xml

```diff
@@ -1575,6 +1589,16 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>jdk11plus</id>
+            <activation>
+                <jdk>[11,)</jdk>
+            </activation>
+            <properties>
+                <failsafe.argLine>--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.net=ALL-UNNAMED</failsafe.argLine>
```

Comment:
As far as I can see, this should not be needed anymore. Even then, we already have this in the guice projects so it should not live here for ALL modules. Only those which need it.